src/routes/boot.ts

```diff
@@ -25,6 +27,17 @@ const excludeProperties = <T, K extends keyof T>(
   return obj;
 };
 
+const getFeatures = async (
+  con: DataSource,
+  userId: string,
+): Promise<IFlags[]> => {
```

Comment:
The return object is not an array but an object where the key is the name of the feature